tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -143,6 +143,18 @@ def _run(self):
                                 ],
                             )
                             NS.volume.save()
+
+                            try:
```

Comment:
Now we are writing the bricks with "append=true". So each brick will have sequence_no as key and this maintains the order of bricks. If we don't delete it every time it gets accumulated.